packages/web-components/src/components/locale-modal/locale-modal.ts

```diff
@@ -187,6 +187,16 @@ class DDSLocaleModal extends DDSExpressiveModal {
     return `${ddsPrefix}-locale-search`;
   }
 
+  /**
+   * A selector selecting the nodes that should be focused when modal gets open.
+   */
+  static get selectorPrimaryFocus() {
+    return `
+      [data-modal-primary-focus],
+      ${ddsPrefix}-region-item
```

Comment:
Thanks for this @asudoh! Can we target the close button instead to be consistent with all our other modals?

```suggestion
      ${ddsPrefix}-expressive-modal-close-button
```